arduino/examples/ESP32IOBoard/ESP32IOBoard.ino

```diff
@@ -344,7 +350,7 @@ void setup()
         new Esp32HardwareCan("esp32can", CAN_RX_PIN, CAN_TX_PIN));
 #endif // USE_CAN
 
-#if defined(USE_WIFI)
+#if defined(USE_WIFI) && defined(BROADCAST_MDNS)
```

Comment:
I would think that MDNS.begin would be inside this if def?

Reply:
It is outside of this section to allow the node's hostname to be registered for lookup. This is also the first step towards an updated version which will connect the client nodes to a hub.

Reply:
So MDNS.begin exports some other service (in addition to the once in this block)?

Reply:
Yes, it starts the mDNS resolver which will be needed for clients to find the hub nodes that are broadcasting.